webview/src/experiments/components/table/messages.ts

```diff
@@ -0,0 +1,16 @@
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
```

Comment:
[Q] Do you think we can consolidate all of the experiments table messages into this file?